recipe-server/normandy/control/templates/control/new.html

```diff
@@ -4,7 +4,7 @@
 <!DOCTYPE html>
 {% get_current_language as LANGUAGE_CODE %}
 
-<html lang="{{ LANGUAGE_CODE|default:"en-us" }}">
+<html lang="{{ LANGUAGE_CODE|default:"en-us" }}" data-csrf="{{csrf_token}}">
```

Comment:
I put that in here to make this PR function before the CSRF fix landed. If this is landing into a branch that already has the CSRF fix then we can drop it.